R/driver-databricks.R

```diff
@@ -33,7 +37,7 @@ NULL
 #'   Specifying these options will disable automated credential discovery.
 #' @param ... Further arguments passed on to [`dbConnect()`].
 #'
-#' @returns An `OdbcConnection` object with an active connection to a Databricks
+#' @return An `OdbcConnection` object with an active connection to a Databricks
```

Comment:
```suggestion
#' @returns An `OdbcConnection` object with an active connection to a Databricks
```

`@return` is superseded in favor of `@returns`.

Reply:
Sweet / reverted.  Thanks